eq-author/src/App/MetadataModal/MetadataTable/Controls/KeySelect.js

```diff
@@ -34,9 +34,14 @@ class KeySelect extends Component {
   };
 
   getChangeValue = changes => {
+    const regex = /^[a-z0-9-_]+$/i;
```

Comment:
Not sure we should have the ignore case flag. Metadata keys should ideally be lower case.

Reply:
Also possibly move creating this regex down to where it's being used in the else if?